verify-go.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+gen_checksum()
+{
+    find ./gen -type f -exec md5sum {} \; | sort -k 2 | md5sum
```

Comment:
This is more portable than `git status` which didn't work with detached HEAD in Travis CI.

Reply:
Can you please add this as a comment in the file?

Reply:
Done, added comments to functions.